### PR TITLE
Auto upgrade firewall

### PR DESCRIPTION
**Pull request by tamirkamara**
## What is being addressed

Since we now support upgrading the template version of deployed resources we should also have the makefile/cicd support upgrading the firewall currently installed in the system.

## How is this addressed

- `devops/scripts/deploy_shared_service.sh` to call `update`

### CODE REVIEW

File: devops/scripts/deploy_shared_service.sh [hunk 1]
```diff
@@ -84,9 +86,19 @@ if ! deploy_result=$(cat << EOF | tre shared-services new --definition-file -
...
-  exit 1
+  ); then
+    echo "Failed to deploy shared service:"
+    echo "${deploy_result}"
+    exit 1
+  fi
+
+else
+
+  echo "An older version is already deloyed. Trying to update..."
```
Comment (LizaShak) [inline devops/scripts/deploy_shared_service.sh:97]: typo in deployed

**Review by anatbal — APPROVED** [review]

File: devops/scripts/deploy_shared_service.sh [hunk 2]
```diff
@@ -59,22 +60,23 @@ if [[ -n "${deployed_shared_service}" ]]; then
...
+    value=${property_values[$index]}
+    additional_props="$additional_props, \"$name\": \"$value\""
+  done
 
-echo "Not currently deployed - deploying..."
-display_name="${template_name#tre-shared-service-}"
-if ! deploy_result=$(cat << EOF | tre shared-services new --definition-file -
+  echo "Not currently deployed - deploying..."
+  display_name="${template_name#tre-shared-service-}"
+  if ! deploy_result=$(cat << EOF | tre shared-services new --definition-file -
```
Comment (LizaShak) [inline devops/scripts/deploy_shared_service.sh:79]: I think the indent here is incorrect

**Review by guybartal — APPROVED** [review]
LGTM

**Review by LizaShak — COMMENTED** [review]
Minor comments

File: devops/scripts/deploy_shared_service.sh (hunk 2 above)
Comment (tamirkamara) [reply devops/scripts/deploy_shared_service.sh:79]: I understand why you think that but due to the EOF is has to be like that.

**Pull request merged**